llvm/lib/Target/BPF/BPFISelLowering.cpp

```diff
@@ -703,6 +715,39 @@ SDValue BPFTargetLowering::LowerSELECT_CC(SDValue Op, SelectionDAG &DAG) const {
   return DAG.getNode(BPFISD::SELECT_CC, DL, VTs, Ops);
 }
 
+SDValue BPFTargetLowering::LowerATOMIC_LOAD(SDValue Op,
+                                            SelectionDAG &DAG) const {
+  const char *Msg =
+      "sequentially consistent (seq_cst) atomic load is not supported";
```

Comment:
Thanks for taking a look at this!
- - -
> Just pass this directly without the message variable?

Then after `git clang-format` it would look like:
```c
  if (cast<AtomicSDNode>(N)->getMergedOrdering() ==
      AtomicOrdering::SequentiallyConsistent)
    fail(DL, DAG,
         "sequentially consistent (seq_cst) atomic store is not supported");
```
I'd prefer keeping it as-is.

Reply:
I wouldn't.
Plus I think such an error should be relegated to an unsupported legalize action.

Reply:
OK, let me change it in the next push.

> Plus I think such an error should be relegated to an unsupported legalize action.

Sorry, could you explain your suggestion a bit more?  Are you suggesting that we add an `Unsupported` enum value to `LegalizeAction`?